packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -203,7 +210,7 @@ function Element({
       onBlur={makeInactive}
       tabIndex={index === 0 ? 0 : -1}
     >
-      <InnerContainer>
+      <InnerContainer $baseColor={backgroundColor} $loaded={loaded}>
```

Comment:
I think the whole logic can be simplified without having to use `onLoad` and `useState`, since we already have direct access to `mediaElement` here.

Can do:

```js
const isLoaded =
    Boolean(mediaElement.current?.complete) ||
    mediaElement.current?.readyState >= 1;
```

And then simply:

```jsx
<InnerContainer $baseColor={!isLoaded && backgroundColor}>
```

Works great in my testing

Reply:
Sadly `isLoaded =
    Boolean(mediaElement.current?.complete) ||
    mediaElement.current?.readyState >= 1;` did work in my testing. Transparent images prove this. 

<img width="363" alt="Screenshot 2021-11-19 at 14 31 08" src="https://user-images.githubusercontent.com/237508/142639433-59f9245d-bc42-425f-8414-60ccd5c0fa60.png">

This google logo as a background, when it should not. When you hover over it, it triggers a repaint. 
See what happens after hovering on the image. 

<img width="284" alt="Screenshot 2021-11-19 at 14 32 48" src="https://user-images.githubusercontent.com/237508/142639661-0fde6b55-f1b2-44df-a441-f20a4b8baffa.png">

I think we should stick with state, as it triggers a repaint.